### PR TITLE
Feature/blog-authors

### PR DESCRIPTION
**Pull request by zysim**
This branch is to solve #12. Stuff I've done include:
- Every blog post made so far should print out the author at the bottom now
- `data.authors` created in `_config.yml` which enables my previous bullet point, plus extra info
- Author's avatar will be replaced by the first letter of their name if we don't have an image

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -63,3 +63,77 @@ exclude:
...
+    sophia:
+      name: Sophia
+      avatar:
+      web:
+      bio: Sophia seeks world domination
+    thomas:
+      name: Thomas
+      avatar:
+      web:
+      bio: Thomas seeks world domination
```
Comment (takluyver) [inline _config.yml:131]: Wow, looks like there's a load of competition for this world domination thing ;-)

**Comment by takluyver** [comment]
This sounds OK to me - I assume you've tested building it and it works as described.

If you've got time to investigate, maybe you could try to fix the netlify builds - it looks like they're running `jekyll serve` when they should be running `jekyll build`. I've no idea where that's controlled from, though.

File: _config.yml (hunk 1 above)
Comment (zysim) [reply _config.yml:131]: Oh wow yeah I totally forgot to remove those default texts haha feel free to edit your own though

**Comment by zysim** [comment]
Aye I'll look into the netlify build when I can. @Nanoseb Ok for me to merge?

**Comment by Nanoseb** [comment]
Arg, really sorry for the delay, I saw it but then forgot about it...
Concerning the bio ;) how visible is the text? is it on every blog post? if so maybe leave it empty for now, and every one can add its own.

**Comment by zysim** [comment]
Updating this PR with a merge from gh-pages, and a new commit, [ba148b](https://github.com/Maritime-Robotics-Student-Society/blog/pull/13/commits/ba148b36c6cf4bdd303caf85c9ea5a4db8872cb0). This new commit allows an optional, article-specific author bio, christened `author_bio` that is placed in the front matter for every article. If it is set, it will be displayed in the bio at the bottom of the article page. This _is_ also meant to default to the bios set in `_config.yml`, but the code for that is currently commented out, until we create decent default bios for everyone. Hence, if `author_bio` isn't set, it'll just display the author's name and nothing else.

**Pull request merged**